File: javascript/troco_verificado.js
const preco = parseFloat(prompt("Digite o preço"));
const qtd = parseInt(prompt("Digite a quantidade"));
const valorRecebido = parseFloat(prompt("Digite o valor recebido"));

const valorTotal = preco * qtd;

if (valorRecebido < valorTotal) {
    console.log("Valor insuficiente, valor restante = R$ %.2f", valorTotal - valorRecebido);
} else {
    let troco = valorRecebido - valorTotal;
    console.log("troco = R$ %.2f", troco);
}

File: python/troco_verificado.py
preco_unitario = float(input("Preço unitário do produto: "))
quantidade = int(input("Quantidade comprada: "))
dinheiro_recebido = float(input("Dinheiro recebido: "))

preco_final = preco_unitario * quantidade

if preco_final > dinheiro_recebido:

    print(f"DINHEIRO INSUFICIENTE. FALTAM {preco_final - dinheiro_recebido:.2f} REAIS")
else:
    troco = dinheiro_recebido - preco_final
    print(f"TROCO = {troco:.2f}")
